org.lflang.tests/src/org/lflang/tests/TestRegistry.java

```diff
@@ -386,32 +386,22 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) {
         @Override
         public FileVisitResult visitFile(Path path, BasicFileAttributes attr) {
             if (attr.isRegularFile() && path.toString().endsWith(".lf")) {
-                // Parse the file. If this is unsuccessful, add the test and
-                // report that it didn't compile.
-                Resource r = rs.getResource(
-                        URI.createFileURI(path.toFile().getAbsolutePath()),
-                        true);
+                // Try to parse the file.
+                Resource r = rs.getResource(URI.createFileURI(path.toFile().getAbsolutePath()),true);
                 // FIXME: issue warning if target doesn't match!
                 LFTest test = new LFTest(target, path);
-                EList<Diagnostic> errors = r.getErrors();
-                if (!errors.isEmpty()) {
-                    for (Diagnostic d : errors) {
-                        test.issues.append(d.toString());
-                    }
-                    test.result = Result.PARSE_FAIL;
```

Comment:
It looks like this functionality was removed without being implemented anywhere else. Any particular reason for this?

Reply:
The parser check was duplicated before. It still exists in `configure()` in `TestBase`. The reason the check is removed here, is that we cannot throw anything except from `IOException` from `visitFile`.